shapeshift/web
# fix: check for undelegations length vs. claim in `<WithdrawCard />`

## Description

We were checking for `hasClaim` vs. `undelegations` length to branch on not/rendering withdraw cards.

The reason why this was never spotted is because claims are very quick to accrue dust. Since live reload was broken for a long time, claiming and reloading the page would accrue enough claim dust for `hasClaim` to be true and undelegations to show. Now that live refetching consistently works, this is an actual bug.

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk



None, if this carries any risks we have bigger problems

## Testing



- Ensure you have at least one pending withdrawal
- Claim your ATOM rewards
- Go back to overview
- Wait for the Tx to go through and the Claim button to be disabled
- Your undelegations should still be visible

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

### Prod

<img width="523" alt="image" src="https://user-images.githubusercontent.com/17035424/218091485-e68aa02b-b742-4869-acbd-2139541143ab.png">

### This diff

<img width="523" alt="image" src="https://user-images.githubusercontent.com/17035424/218091485-e68aa02b-b742-4869-acbd-2139541143ab.png">
